.helm/ecamp3/templates/hook_db_drop.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Automatically drops the database in an external PostgreSQL when completely uninstalling the chart.
+#
+{{- if and (not .Values.postgresql.enabled) .Values.postgresql.dropDBOnUninstall }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ include "app.name" . }}-db-drop"
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ include "app.name" . }}-post-delete"
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: db-drop-job
+          image: "{{ .Values.php.image.repository }}:{{ .Values.php.image.tag | default .Values.imageTag }}"
+          command: ["php", "bin/console", "doctrine:database:drop", "--if-exists", "-f"]
```

Comment:
I think we should be super careful we don't accidentally remove `dev` or even `stage` or `prod` by some wrong configuration. Any chance we could double check here on these values?

Reply:
Sure, I will have to extract the database name to a separate config value anyways, due to your finding with the postgres_admin_url in the logs. So if you want to pay the price of putting knowledge about specific environments such as `dev`, `stage` or `prod` in here, we can prevent them from ever being deleted. I'll have a look how we could most easily do this.

An easy solution would probably be to extend the big `{{- if }}` at the top of this file to also check that we're not in `dev`, `stage` or `prod`.

Reply:
But keep in mind that we are in a generic helm chart here, so if we hardcode `dev` to be undeletable, no one using the helm chart will be able to delete their `dev` database. So it's a tradeoff between configurability and safety...

Reply:
Another option would be to double check in the to-uninstall step (continuous-deployement.yml), that the list doesn't contain certain branches. Then we could keep the helm charts clean.

Reply:
That sounds like a good idea. If we ever really want to delete dev, we can also do it manually.

And I could also `--set postgres.dropDBOnUninstall=false` to false on special branches and `true` on feature branch deployments. I just noticed I forgot to set the flag at all so far, so the DB currently is never deleted, not even for feature branches.